platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/maps/NativeMapView.java

```diff
@@ -1048,9 +1048,13 @@ protected void onSnapshotReady(@Nullable Bitmap mapContent) {
       return;
     }
 
-    Bitmap viewContent = viewCallback.getViewContent();
-    if (snapshotReadyCallback != null && mapContent != null && viewContent != null) {
-      snapshotReadyCallback.onSnapshotReady(BitmapUtils.mergeBitmap(mapContent, viewContent));
+    if (viewCallback == null && snapshotReadyCallback != null && mapContent != null) {
```

Comment:
NIT `snapshotReadyCallback` and `mapContent` checks can wrap the `viewCallback` check